dask/dataframe/methods.py

```diff
@@ -458,11 +460,13 @@ def monotonic_increasing_aggregate(concatenated):
 
 
 def monotonic_decreasing_chunk(x):
-    data = x if is_index_like(x) else x.iloc
-    return pd.DataFrame(
-        data=[[x.is_monotonic_decreasing, data[0], data[-1]]],
-        columns=["monotonic", "first", "last"],
-    )
+    if len(x) == 0:
```

Comment:
Similar comment here

```suggestion
    if x.empty:
```